### DISCUSSION TITLE
How to Build slim with exclude package python

### DISCUSSION THREAD

**Discussion by nvlrmdn** [slimtoolkit/slim#683] [Q&A]
i have the image that have size 8gb and i want to compress the size of image with slim but if i use the compressed image the container is not want to running because the image dont have the package he need, i run the service in python script and the package i need is torch, how to i keep the package torch when i compress the image with slim ?
the command i use is like this :
`slim build --target the_8gb_image --tag tagcompressed:slim --http-probe=false`

**Comment by kcq** [comment]
@nvlrmdn The best use case for the app is when you have a full app in the container, which you may or may not have in your case (not sure here... you mentioned you have a python script, but does it use all packages you need?). Minifying base images is more involved because you need to explicitly specify everything you need (or most of it). There's no simple way to remove specific system or python packages you don't need at this point in time.

In your case, you might need to enhance the script you have to use more python dependencies. Either way, take a look at the `--include-path` and other `--include-*` flags (e.g., `--include-bin`, etc). These flags are used to explicitly add extra directories or files to the minified images in addition to what's included automatically. For node.js applications there's also the `--include-node-package` flag that allows you to keep additional node.js packages by name.
